### PR TITLE
Bump to v3.5, update format

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@xmnlab and @conda-forge/core there is an automatically generated PR that's been sitting there but unable to be auto-merged. I've tried to manually bump the version here as I have another package that needs v3.5.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
   home: https://github.com/henriquebastos/python-decouple/
   license: MIT
   license_family: MIT
-  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
+  license_file: LICENSE
   summary: Strict separation of config from code.
   description: |
     Decouple helps you to organize your settings so that you can change 
     parameters without having to redeploy your app.
-  doc_url: https://github.com/henriquebastos/python-decouple/
```
Comment (xmnlab) [inline recipe/meta.yaml:38]: Is there any reason for removing that?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
-  dev_url: https://github.com/henriquebastos/python-decouple/
```
Comment (xmnlab) [inline recipe/meta.yaml:39]: Same here

**Review by xmnlab — COMMENTED** [review]
Lgtm. Just added 2 question in the comments

**Comment by thewchan** [comment]
@xmnlab Is this good to go?

**Comment by xmnlab** [comment]
@thewchan I added 2 comments https://github.com/conda-forge/python-decouple-feedstock/pull/8#pullrequestreview-860211836

any reason for removing doc_url and dev_url ?

other than that, looks good to me

**Comment by thewchan** [comment]
> @thewchan I added 2 comments [#8 (review)](https://github.com/conda-forge/python-decouple-feedstock/pull/8#pullrequestreview-860211836)
> 
> any reason for removing doc_url and dev_url ?
> 
> other than that, looks good to me

I mentioned above that the same url is already in `home:`, seems redundant to list them again.

**Comment by xmnlab** [comment]
> > @thewchan I added 2 comments [#8 (review)](https://github.com/conda-forge/python-decouple-feedstock/pull/8#pullrequestreview-860211836)
> > any reason for removing doc_url and dev_url ?
> > other than that, looks good to me
> 
> I mentioned above that the same url is already in `home:`, seems redundant to list them again.

for some reason, I am not able to see your comment about that ...

I hear, and I understand your point. but I think that it is not redundant (although it looks redundant) ... if the user wants to know "what is the doc url" ... it is preferable to have the same link as used by home than have no information. 

please keep both parameters dev_url and doc_url.

if you want to can change the doc_url to `https://github.com/henriquebastos/python-decouple/blob/master/README.rst`

thank you so much for updating the recipe and adding yourself as a maintainer, any help is very appreciated!

**Comment by thewchan** [comment]
sure I can do that

**Comment by xmnlab** [comment]
@thewchan thank you so much! merged! :)

**Pull request merged**